src/store/user.js

```diff
@@ -77,7 +77,8 @@ const actions = {
                 context.commit('SET_CAPABILITIES', cap)
                 context.commit('SET_USER', {
                   id: res.id,
-                  displayname: res['display-name'],
+                  username: res.username,
+                  displayname: res.displayname || res['display-name'],
```

Comment:
seems you also fixed display name here but didn't mention it in changelog

Reply:
A forgot that one - added an item now :wink: